### DISCUSSION TITLE
Brew and casks are not upgrading/updating

### DISCUSSION THREAD

**Discussion by hichrisbell** [Homebrew/discussions#3365] [Casks]
I have run `brew upgrade` as well as `brew cu -a` to initiate an update of brew and the casks I have installed.

Everything is reporting as current with no updates pending. However, when I check the pull requests and run a livecheck (i.e. `brew livecheck --all --cask --newer-only`), I can see there are many casks which have been updated, but the data isn't flowing to my Mac.

For example, I have

- expressvpn 10.19.1.65321 as current and latest, and the repo was updated six days ago with 11.1.1.66865.
- zoom 5.10.6.7523 as current and latest, and the repo was updated 16 days ago with 5.10.7.7748

What are my next steps? Do I need to do a complete uninstall/reinstall?

**Comment by carlocab** [comment]
What is the output of `brew config` and `brew doctor`?

**Reply by hichrisbell** [reply]
I found the issue thanks to brew doctor. Thanks for the prompt.
